src/components/ProgramRecord/RecordsHelp.jsx

```diff
@@ -15,20 +15,20 @@ const RecordsHelp = ({ helpUrl }) => (
     <p>
       <FormattedMessage
         id="help.section.content.with.link"
-        defaultMessage="To learn more about records you can "
+        defaultMessage="To learn more about records you can {link}"
         description="Content for the help section with a link to the records help area"
+        values={{
+          link: (
+            <Hyperlink
+              destination={helpUrl}
+              target="_blank"
+              showLaunchIcon={false}
+            >
+              read more in our records help area.
```

Comment:
is this second section hard-coded english?

Reply:
This was brought up in a conversation with @Tj-Tracy. He informed me that we are still on `react-intl@2.9.0` because of constraints with how we use proptypes. Check out [this link](https://formatjs.io/docs/react-intl/upgrade-guide-3x/#enhanced-formattedmessage--formatmessage-rich-text-formatting) to examples I followed for embedding links in a `defaultMessage` prop. He also mentioned that we should maybe consider migrating to v3 of `formatjs`.

I was under the impression that this works as intended. The text all ends up inside the `defaultMessage` and `react-intl` is smart enough to parse out the HTML tags. I agree that it looks a little off, though.

Reply:
Ah, thanks for the explanation Max.